### PR TITLE
Starting on licenses.

### PR DESCRIPTION
**Pull request by vonkow**
Not my favorite subject, by a long shot.

### CODE REVIEW

**Comment by vonkow** [comment]
Still need to carve out Stories from both product identity and OGC? 

Is there an even less restrictive way of expressing "don't steal it wholesale and say you made it but please take it and make it yours unless you're doing something evil like promoting racism" than the current Product Identity + very permissive 3rd-party license strategy?

File: LICENSE-mit.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright 2020 (TBD)
```
Comment (sarah-johnson) [inline LICENSE-mit.md:1]: Replace TBD

File: README.md [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+
+The" Game Content" refers to any images and written copy in this project, including all files written in Markdown. 
+
+### Written Content
+
+The Written portions of the Game Content that are not explicitly Product Identity are licenced under the OPEN GAME LICENSE Version 1.0a. The licence and declaration of what is and isn't Product Identity can be found in the [LICENSE-ogl.md](LICENSE-ogl.md) file.
+
+### Images
+
+Some images in this project are licensed, others are in the Public Domain. See [LICENSE-images.md](LICENSE-images.md) for these licenses.
```
Comment (sarah-johnson) [inline README.md:36]: Still need to check this in

**Comment by sarah-johnson** [comment]
https://github.com/karst-club/karst-gatsby/pull/11

**Pull request closed without merging**